src/partners/lifi.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  asArray,
+  asMaybe,
+  asNumber,
+  asObject,
+  asOptional,
+  asString,
+  asValue
+} from 'cleaners'
+
+import {
+  asStandardPluginParams,
+  PartnerPlugin,
+  PluginParams,
+  PluginResult,
+  StandardTx,
+  Status
+} from '../types'
+import { datelog, retryFetch, smartIsoDateFromTimestamp, snooze } from '../util'
+
+const asStatuses = asMaybe(asValue('DONE'), 'other')
+const asToken = asObject({
+  // address: asString,
+  // chainId: asNumber,
+  symbol: asString,
+  decimals: asNumber
+  // name: asString,
+  // coinKey: asString,
+  // logoURI: asString,
+  // priceUSD: asString
+})
+
+const asTransaction = asObject({
+  txHash: asString,
+  // txLink: asString,
+  amount: asString,
+  token: asToken,
+  // chainId: asNumber,
+  // gasPrice: asString,
+  // gasUsed: asString,
+  // gasToken: asToken,
+  // gasAmount: asString,
+  // gasAmountUSD: asString,
+  amountUSD: asOptional(asString),
+  // value: asString,
+  timestamp: asOptional(asNumber)
+})
+
+const asTransfer = asObject({
+  // transactionId: asString,
+  sending: asTransaction,
+  receiving: asTransaction,
+  // lifiExplorerLink: asString,
+  // fromAddress: asString,
+  toAddress: asString,
+  // tool: asString,
+  status: asString
+  // substatus: asString,
+  // substatusMessage: asString,
+  // metadata: asObject({
+  //   integrator: asString
+  // })
+})
+
+// Define the cleaner for the whole JSON
+const asTransfersResult = asObject({
+  transfers: asArray(asTransfer)
+})
+
+type Transfer = ReturnType<typeof asTransfer>
```

Comment:
Type never used.

Reply:
Weird that lint didn't catch that. fixed